lighthouse-core/lib/stack-collector.js

```diff
@@ -77,7 +77,7 @@ async function collectStacks(passContext) {
   })()`;
 
   /** @type {JSLibrary[]} */
-  const jsLibraries = await passContext.driver.evaluateAsync(expression);
+  const jsLibraries = await passContext.driver.evaluateAsync(expression, {useIsolation: true});
```

Comment:
we can't move this one to isolation unfortunately, the detector requires the same global scope to detect some of its libraries :/

Reply:
should probably get a smoketest to catch this if there isn't one already

Reply:
Hm. Well that's unfortunate.

tbh I don't know, is it just bad practice to intercept `fetch()` and error it, like what was happening in #10065? Bad enough that we could audit for it and bail fatally instead of isolate?

Reply:
> should probably get a smoketest to catch this if there isn't one already

 there is one